### PR TITLE
Improve code quality for test cases for EditLabCommand

### LINKED ISSUE #444: Improve the code quality of EditLabCommandTest
- Instead of using raw values, use references.

### PR DESCRIPTION
**Pull request by reddevil1313**
- Fixes #444

### CODE REVIEW

File: src/test/java/seedu/programmer/storage/JsonProgrammerErrorStorageTest.java [hunk 1]
```diff
@@ -96,10 +96,10 @@ public void saveProgrammerError_nullProgrammerError_throwsNullPointerException()
     /**
      * Saves {@code ProgrammerError} at the specified {@code filePath}.
```
Comment (shermannws) [inline src/test/java/seedu/programmer/storage/JsonProgrammerErrorStorageTest.java:97]: same, javadoc should be refactored similarly

Comment (reddevil1313) [reply src/test/java/seedu/programmer/storage/JsonProgrammerErrorStorageTest.java:97]: Yeap, makes sense!

File: src/main/java/seedu/programmer/logic/commands/EditLabCommand.java [hunk 2]
```diff
@@ -47,6 +47,7 @@
      * @param original the lab to be edited.
```
Comment (shermannws) [inline src/main/java/seedu/programmer/logic/commands/EditLabCommand.java:47]: Incomplete @param tags for javadocs

File: src/main/java/seedu/programmer/logic/commands/EditLabCommand.java [hunk 3]
```diff
@@ -55,6 +56,7 @@ public EditLabCommand(Lab original, LabNum newLabNum, LabTotal total) {
...
```
Comment (shermannws) [inline src/main/java/seedu/programmer/logic/commands/EditLabCommand.java:56]: javadocs here too

**Review by shermannws — CHANGES_REQUESTED** [review]
Some javadocs incompleteness and inconsistency to be fixed before approval! Thanks a lot!

File: src/main/java/seedu/programmer/logic/commands/EditLabCommand.java (hunk 2 above)
Comment (reddevil1313) [reply src/main/java/seedu/programmer/logic/commands/EditLabCommand.java:47]: Done!

File: src/main/java/seedu/programmer/logic/commands/EditLabCommand.java (hunk 3 above)
Comment (reddevil1313) [reply src/main/java/seedu/programmer/logic/commands/EditLabCommand.java:58]: Done!

**Review by shermannws — APPROVED** [review]
LGTM

**Pull request merged**